README.md

```diff
@@ -179,6 +179,10 @@ a suggestion is attempted to be highlighted either by the keyboard or mouse. It
 scope, access to an object called `suggestion` which is the current suggestion that is being
 interacted with. A non-selectable suggestion cannot be clicked on, hovered over, or interacted with
 via the keyboard.
+ - `inputHint({query}) {Promise}`: An expression that hints at a suggestion by resolving a
```

Comment:
I guess this makes sense, but leaves out the possibility of hitting two separate endpoints for a hint vs a suggestion. A single hint could also potentially be a much faster endpoint hit, or maybe not even a remote call at all, while suggestions would be slower.

Do you think it's ok to not support that use case?